Update unity quick start basing on sdk 1.0.9. (#225)
easemob/easemob-doc: docs/document/unity/quickstart.md

   2. 在弹出的 **Select Editor version and platform** 窗口中，选择本地安装的 Editor 版本，并根据后续提示打开项目。
   :::
 
   :::notice

>    如果碰到'SpriteRenderer' does not contain a definition for 'IsUsingDeformableBuffer'这类异常，需要进行reset操作。

Reviewer: ```suggestion
   如果遇到 'SpriteRenderer' does not contain a definition for 'IsUsingDeformableBuffer' 这类异常，需进行重置操作。
```

Author: ok

>    **Help**菜单下，执行**Reset Packages to defaults**。

Reviewer: 是下面这个意思吗？

在 **Help** 菜单下，点击 **Reset Packages to defaults** 将包恢复默认设置。

   :::

### 2. 集成环信即时通讯 SDK

你可以参考以下步骤集成 SDK：

1. [下载 Unity SDK](https://downloadsdk.easemob.com/downloads/SDK/Unity/agora_chat_unity_sdk1.0.9.unitypackage)。

> 2. 在 Unity Editor 中，选择 **Assets > Import Package > Custom Package...**，然后选择刚下载的unitypackage导入。

Reviewer: ```suggestion
2. 在 Unity Editor 中，选择 **Assets > Import Package > Custom Package...**，然后选择刚下载的 unitypackage 导入。
```

Reviewer: 这里的unitypackage要改为“Unity 打包文件”？

Author: 这里的unitypackage要改为“Unity 打包文件”？ --- 这个不需要
unitypackage前后的空格已经加上。

在 **TestCode.cs** 头部添加以下命名空间：

```csharp
using AgoraChat;
using AgoraChat.MessageBody;
```

### 3. 初始化 SDK

在 `InitSDK` 方法中添加以下代码完成 SDK 初始化：

```csharp

> var options = new Options("appkey"); //参数为你的appkey

Reviewer: ```suggestion
var options = new Options("appkey"); //将该参数设置为你的 App Key
```

Author: ok
